apps/storefront/src/pages/QuickOrder/components/QuickAdd.tsx

```diff
@@ -179,7 +179,7 @@ export default function QuickAdd(props: AddToListContentProps) {
         return;
       }
 
-      if (isStock === '1' && quantity > Number(stock)) {
+      if (isStock === '1' && allQuantity > Number(stock)) {
```

Comment:
This is the only real behaviour change in this PR. If you look a few lines below, we use `allQuantity` to compute whether min/max have been reached. I'm guessing we need to do the same for checking if the stock has been exceeded. cc: @bc-micah